Cargo.toml

```diff
@@ -36,7 +36,7 @@ product-name = "Space Acres"
 
 [dependencies]
 anyhow = "1.0.80"
-betrayer = { git = "https://github.com/hunjixin/betrayer.git", rev="bf5e5a4" }                             
+betrayer = { git = "https://github.com/sidit77/betrayer.git", rev="5f5c4bc" }                             
```

Comment:
This should use full hash and have a TODO saying which upstream version to upgrade to to switch to upstream since this is a temporary solution. Also there are still a lot of whitespaces at the end of this line unnecessarily.

Reply:
comment on cargo.toml?

Reply:
Example of actionable TODOs that were eventually removed: https://github.com/subspace/subspace/pull/2587